make-sync-function

```diff
@@ -42,30 +41,31 @@ try {
   // which it will be injected
   syncDocDefn = syncDocDefn.trim().replace(/(?:\r\n)|(?:\r)|(?:\n)/g, function() { return '\n  '; });
 } catch (ex) {
-  console.log('Unable to read the sync document definitions file: ' + ex);
-
-  return errorStatus;
+  if (ex.code === 'ENOENT') {
+    console.log('ERROR: Sync document definitions file does not exist');
+  } else {
+    console.log('ERROR: Unable to read the sync document definitions file: ' + ex);
+  }
+  process.exit(errorStatus);
 }
 
 var syncFuncTemplate;
 try {
   syncFuncTemplate = fs.readFileSync(scriptDir + '/etc/sync-function-template.js', 'utf8');
 } catch (ex) {
-  console.log('Unable to read the sync function template file: ' + ex);
+  console.log('ERROR: Unable to read the sync function template file: ' + ex);
 
-  return errorStatus;
+  process.exit(errorStatus);
 }
 
 var syncFunc = syncFuncTemplate.replace('%SYNC_DOCUMENT_DEFINITIONS%', function() { return syncDocDefn; });
 
 try {
   fs.writeFileSync(outputFilename, syncFunc, 'utf8');
 } catch (ex) {
-  console.log('Unable to write the sync function to the output file: ' + ex);
+  console.log('ERROR: Unable to write the sync function to the output file: ' + ex);
 
-  return errorStatus;
+  process.exit(errorStatus);
 }
 
 console.log('Sync function written to ' + outputFilename);
-
-return successStatus;
```

Comment:
This isn't necessary (or effective). The process will return an exit status of zero if it exits normally anyway.